kymatio/scattering1d/frontend/sklearn_frontend.py

```diff
@@ -11,4 +11,12 @@ class ScatteringTransformer1D(ScatteringTransformerMixin, ScatteringNumPy1D):
 ScatteringTransformer1D._document()
 
 
-__all__ = ['ScatteringTransformer1D']
+class TimeFrequencyScatteringTransformer1D(ScatteringTransformerMixin,
+                                           ScatteringNumPy1D):
+    pass
```

Comment:
This doesn't seem to contain anything about time frequency scattering

Reply:
needed for docs